contracts/zaps/OlympusZapV2.sol

```diff
@@ -78,7 +78,7 @@ contract Olympus_V2_Zap_In is ZapBaseV3 {
         uint256 toInvest = _pullTokens(fromToken, amountIn, referral, true);
 
         // approve "swapTarget" to spend this contracts "fromToken" if needed
-        _approveToken(fromToken, swapTarget, toInvest);
+        //_approveToken(fromToken, swapTarget, toInvest);
```

Comment:
You can remove this line instead of commenting it out.

Reply:
Comments removed

Reply:
Comments removed